src/notte/agents/main.py

```diff
@@ -56,3 +57,41 @@ async def async_run(self, task: str, url: str | None = None) -> AgentResponse:
     def run(self, task: str, url: str | None = None) -> AgentResponse:
         agent = self.create_agent()
         return asyncio.run(agent.run(task, url=url))
+
+    # Add to the relevant section that handles tasks
+    async def handle_arxiv_task(self, task: str, url: str) -> str:
+        """Handle ArXiv specific tasks."""
+        if "arxiv.org" in url and "figures" in task.lower() and "tables" in task.lower():
+            # Extract paper title from task
+            paper_title_match = re.search(r'"([^"]+)"', task)
+            if not paper_title_match:
+                return "Could not identify paper title in task."
+            
+            paper_title = paper_title_match.group(1)
+            
+            # First navigate to ArXiv
+            await self.act(GotoAction(url=url))
+            
+            # Search for the paper
+            search_box = self.env.snapshot.dom_node.find("I1")  # Assuming I1 is the search box ID
+            if search_box:
+                await self.act(FillAction(id="I1", value=paper_title))
```

Comment:
There is no `act` method on the `Agent` class, this cannot work. Have you tested it before actually creating the PR ?